daos/mysql/Items.php

```diff
@@ -247,16 +291,16 @@ public function get($options = array()) {
         // first check whether more items are available
         $result = \F3::get('db')->exec('SELECT items.id
                    FROM '.\F3::get('db_prefix').'items AS items, '.\F3::get('db_prefix').'sources AS sources
-                   WHERE items.source=sources.id '.$where.' 
+                   WHERE items.source=sources.id AND '.$where_sql.'
                    LIMIT 1 OFFSET ' . ($options['offset']+$options['items']), $params);
         $this->hasMore = count($result);
 
         // get items from database
-        return \F3::get('db')->exec('SELECT 
+        return \F3::get('db')->exec('SELECT
                     items.id, datetime, items.title AS title, content, unread, starred, source, thumbnail, icon, uid, link, updatetime, author, sources.title as sourcetitle, sources.tags as tags
                    FROM '.\F3::get('db_prefix').'items AS items, '.\F3::get('db_prefix').'sources AS sources
-                   WHERE items.source=sources.id '.$where.' 
-                   ORDER BY items.datetime '.$order.' 
+                   WHERE items.source=sources.id AND '.$where_sql.'
+                   ORDER BY items.datetime '.$order.'
```

Comment:
Actually they are only ordered by `datetime`, this should be also ordered by `id` for seek pagination to work properly.